portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,12 +27,21 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private ArrayList<String> comments = new ArrayList<String>( 
-    Arrays.asList("Comment 1", "Comment 2", "Comment 3"));
+    Arrays.asList());
```

Comment:
Nit: No need for Arrays.asList. `private ArrayList<String> comments = new ArrayList<String>();` is enough to create an empty ArryayList.

Reply:
Remember to remove the importer Arrays since it will be unused after you delete this line.

Reply:
Nice catch! Arrays has been removed.